performance/tests/ListItemsTests.js

```diff
@@ -0,0 +1,135 @@
+/* global page, minFPS, maxFID, maxCLS, stepNumber, testMultiple, maxDCL, maxFCP, maxLCP, passRatio */
+/* eslint-disable*/
+const TestResults = require('../TestResults');
+const {CLS, FID, FPS, getAverageFPS, PageLoadingMetrics} = require('../TraceModel');
+const {clsValue, firstInputValue, getFileName, scrollAtPoint} = require('../utils');
+
+const listItemTests = (componentName, dataSize, listID, itemID, outputFileName) => describe(componentName, () => { // eslint-disable-line
```

Comment:
Becase there is many param, you would better to use object .

```suggestion
const listItemTests = ({component, dataSize, listId, itemId, outputFileName}) => describe(componentName, () => { // eslint-disable-line
```
And on the test.js

const component = 'Items'
listItemTests({
component, 
dataSize: 100, 
listId: component,
itemId: 'item', 
outputFileName: component 
});
is more clear and readable.

I guess `itemId` can be removed
and `listId` and `outputFileName` can have default value from  `componentName`.
than, 

listItemTests({component, dataSize: 100});

or 

listItemTests(component, 100); possible.